tests/nightly/test_large_array.py

```diff
@@ -279,6 +279,19 @@ def test_diag():
     assert_almost_equal(r.asnumpy(), np.diag(a_np, k=k))
 
 
+def test_ravel_and_unravel():
```

Comment:
break it into two tests

Reply:
Its more readable and understandable this way. Also I have split it into 2 functions inside.